http/src/main/resources/swagger/api-docs.yaml

```diff
@@ -1320,14 +1320,14 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/ErrorReport"
-        "420":
-          description: Runtime not ready
+        "422":
```

Comment:
think we should bump the swagger version

Reply:
Will do!